opengever/maintenance/scripts/list_decided_proposals_without_excerpt.py

```diff
@@ -0,0 +1,72 @@
+"""
+Searches for all resolved proposals with no excerpt and lists them.
+
+    bin/instance run ./scripts/list_decided_proposals_without_excerpt.py
+
+"""
+from opengever.maintenance.debughelpers import setup_app
+from opengever.maintenance.debughelpers import setup_option_parser
+from opengever.maintenance.debughelpers import setup_plone
+from opengever.maintenance.utils import LogFilePathFinder
+from opengever.maintenance.utils import TextTable
+import sys
+
+from opengever.meeting.model.proposal import Proposal
+
+
+class DecidedProposalWithoutExcerptLister(object):
+
+    def __init__(self):
+        self.table = TextTable(col_max_width=60)
+        self.table.add_row(["proposal path", "proposal title"])
+
+    def list_proposals_without_excerpt(self):
+        for proposal in self.get_proposals_without_excerpt():
+            proposal.agenda_item
```

Comment:
these statements do nothing and seem like a leftover.

Reply:
Oh I'm sorry, that looks like a dirty leftover...